### ISSUE TITLE
New Term Request for Parasitemia

### ISSUE THREAD

**Issue by lnanderson** [obophenotype/human-phenotype-ontology#10138]
**Preferred term label:** Parasitemia

**Synonyms:** parasitemia, parasitemias, blood parasite detection

**Definition (free text, please give PubMed ID):** is the quantitative detection of parasites in the bloodstream.

**Source Reference(s):** 
- Source Book Title: Principles and Practice of Pediatric Infectious Diseases, 6th Edition (2023); ISBN:978-0-323-75608-2 (Chapters 258, 271, 275, 273, 57, 296)
- Source DOI: https://doi.org/10.1016/C2019-0-00075-1
- Source URL: https://www.sciencedirect.com/topics/medicine-and-dentistry/parasitemia 

**Parent term:** https://hpo.jax.org/app/browse/term/HP:0031863

**Suggested Child term(s):** https://hpo.jax.org/app/browse/term/HP:0034523

**Diseases characterized by this term? (e.g. Orphanet or OMIM number)** [OMIM:248310](http://purl.bioontology.org/ontology/OMIM/248310)

**Related dbXrefs:** [MONDO:0024314](http://purl.obolibrary.org/obo/MONDO_0024314), [MESH:D018512](http://purl.bioontology.org/ontology/MESH/D018512), [MedGen_UID:116650](https://www.ncbi.nlm.nih.gov/medgen/116650), [UMLS:C0242723](http://linkedlifedata.com/resource/umls/id/C0242723), [NCIT:C120903](http://purl.obolibrary.org/obo/NCIT_C120903)

**Your nano-attribution (ORCID)** [0000-0002-8741-7823](https://orcid.org/0000-0002-8741-7823)

**Comment by lnanderson** [comment]
@pnrobinson - new term request is representative of a more general term, similar to existing labels for [Viremia](https://hpo.jax.org/app/browse/term/HP:0020071) and [Bacteremia](https://hpo.jax.org/app/browse/term/HP:0031864), for describing disease-based mechanisms of detection in the host. The existing term for [Blood parasite](https://hpo.jax.org/app/browse/term/HP:0034523) seemed more suitable as subclass to the suggested new term (rather than a synonym) as it lists specific parasites in the description and does not reference the quantitative context. Let me know if this fits the scope, otherwise I can request it as a synonym instead.

Thanks, I appreciate your time!

**Comment by pnrobinson** [comment]
Thanks!
[Blood parasite HP:0034523](https://hpo.jax.org/app/browse/term/HP:0034523) is intended to be synonymous with Parasitemia -- I added this synonym. Parasitemia has specific sub terms, but if an individual has any of the this sub terms, they also have Parasitemia/Blood parasite.

**Comment by lnanderson** [comment]
thank you @pnrobinson, much appreciated!
